compiler-cli/src/lsp.rs

```diff
@@ -7,6 +7,7 @@ use gleam_core::{
 };
 
 pub fn main() -> Result<()> {
+    eprintln!("Starting language server...");
```

Comment:
I didn't see this macro used anywhere else in the project which made me a bit skeptical this is the right call, but I am also not familiar enough with Rust to decide on something different.